doc/how_to_uikeys.md

```diff
@@ -0,0 +1,12 @@
+How to configure UIKeys
+=======================
+
+Disable the BAR Hotkeys widget, press F11 in game and click on the BAR Hotkeys entry, make sure to make it from green to red.
+
+Copy the default uikeys sample from [here](uikeys_default.txt) and paste it in a file called `uikeys.txt` in your BAR root folder.
+Use [this file](uikeys_gridmenu.txt) instead if you use gridmenu.
+
```

Comment:
I think ideally there would be a basic explanation (potentially a link to another document/wikipage?) of the uikeys entries themselves since these can get confusing.

For example:
 * `fakemeta  0x020`. What is fake meta?
 * 
```
bind            Shift+Z  buildunit_armmex      //  - armmex
bind            Shift+Z  buildunit_armamex     //  - armamex
bind            Shift+Z  buildunit_cormex      //  - cormex
bind            Shift+Z  buildunit_corexp      //  - corexp
```
How do multiple bindings work? What are the names like `armamex` and where do I get them from? What if I have an Cor constructor selected and the first binding is an Arm unit (or T1 con and T2 stuff is listed, etc)?

 * `bind                ^,^  drawlabel` What are the mechanics of multi-key bindings? What happens if I also have single `^` bound, does it get triggered too? What if I want to trigger single `^` twice? How do I bind something to a double-tap of the comma key?
 * `bind             Ctrl+B  select AllMap+_Builder_Idle+_ClearSelection_SelectOne+` What is this magic incantation? Where do I find the possible commands (also more "obvious" stuff lke `buildspacing inc`)?